src/components/LoginForm.tsx

```diff
@@ -0,0 +1,74 @@
+
+import * as React from 'react';
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
```

Comment:
Interesting moment. You are using this library, but do you need it? Please, check it via smaller devices, because with provided by you video, it is not obviously

Reply:
I didn't check it on smaller screens, but i noticed, when i try to use keyboard, all screen moves to up.
Do i need to use another way to fix it?